### PR TITLE
Update ckan.po

### LINKED ISSUE #5115: French: "alphanumeric characters" translated to "lettres minuscules, des chiffres"
The English string

> Tag \"%s\" must be alphanumeric characters or symbols: -_.

...which is used in ckan/logic/validators.py:428 to validate tags, has an inaccurate translation in French:

> Le mot-clé \"%s\" ne peut contenir que des lettres minuscules, des chiffres ou les symboles : -_.

An alphanumeric character can be a digit or any letter, whether lowercase or not, so the French description is overly restrictive. An accurate translation could therefore be:
"Le mot-clé \"%s\" ne peut contenir que des lettres, des chiffres ou les symboles : -_."

This affects CKAN 2.4.1, and persists in the master branch as of a3a6b582fcd758727c98b628d13d198b4317787c.

Note that it is unclear which (special) symbols these strings refer to.

### PR DESCRIPTION
**Pull request by Gauravp-NEC**
fix for issue 5115

Fixes #5115 

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by Gauravp-NEC** [comment]
@amercader , Please review this PR. Thankyou

File: ckan/i18n/fr/LC_MESSAGES/ckan.po [hunk 1]
```diff
@@ -1720,10 +1720,10 @@ msgstr "La longueur du mot-clé \"%s\" est supérieure à la longueur maximale %
```
Comment (amercader) [inline ckan/i18n/fr/LC_MESSAGES/ckan.po:1720]: Please revert changes on this file. po files are managed on Transifex, and will get updated by translators once the base English string is updated

File: ckan/logic/validators.py [hunk 2]
```diff
@@ -454,7 +454,7 @@ def tag_name_validator(value, context):
 
     tagname_match = re.compile('[\w \-.]*$', re.UNICODE)
     if not tagname_match.match(value):
-        raise Invalid(_('Tag "%s" must be alphanumeric '
+        raise Invalid(_('Tag "%s" must consist of alphanumeric '
                         'characters or symbols: -_.') % (value))
```
Comment (amercader) [inline ckan/logic/validators.py:458]: @Chealer do you like this message?

> Tag "%s" can only contain alphanumeric characters, spaces (" "), hyphens ("-"), underscores ("_") or dots (".")

```suggestion
        raise Invalid(_('Tag "%s" can only contain alphanumeric '
                        'characters, spaces (" "), hyphens ("-"), '
                        'underscores ("_") or dots (".")') % (value))
                  
                        
```

**Comment by Gauravp-NEC** [comment]
@amercader , I have revert the changes in the .po file in commits https://github.com/ckan/ckan/pull/5760/commits/4dad833ed60a9b1e460b830f5e5ac3161387edb9 and https://github.com/ckan/ckan/pull/5760/commits/92af6b14ee25a4b7eebfd528a055a3ed0eac7b1a. I have also commit your suggestion in https://github.com/ckan/ckan/pull/5760/commits/46e37544da99e085b1568ec13b1845e856107f73. Please merge this PR if it looks good. Thankyou :)

File: ckan/logic/validators.py (hunk 2 above)
Comment (Chealer) [reply ckan/logic/validators.py]: Thank you @amercader and Gauravp-NEC, I like it a lot, but:

- English is not my native language. I am not fully qualified to confirm the phrasing. English **might** have a more precise/correct name for the "." character than "dot".
- As for the meaning, I doubt that underscores are supported. Your 2019-12-16 comment implied they were, but my reading of current [master's tag_name_validator()](https://github.com/ckan/ckan/blob/master/ckan/logic/validators.py#L455) is that they are not. That being said, Python regular expressions are definitely not my native language neither.

**Comment by Gauravp-NEC** [comment]
@amercader @Chealer  ,Did the below changes looks better than commit https://github.com/ckan/ckan/pull/5760/commits/46e37544da99e085b1568ec13b1845e856107f73 ? If you agree I will commit the changes again. Please let me know your opinion. Thankyou :)

image

**Comment by Chealer** [comment]
@Gauravp-NEC I believe the previous version was acceptable, but:

- I agree that "and" must be preferable to "or".
- I am not qualified to confirm that "period" is better than "dot", but it probably is.

However, "period" would need an "s" to remain plural.

**Comment by Gauravp-NEC** [comment]
> @Gauravp-NEC I believe the previous version was acceptable, but:
> 
> * I agree that "and" must be preferable to "or".
> * I am not qualified to confirm that "period" is better than "dot", but it probably is.
> 
> However, "period" would need an "s" to remain plural.

@amercader , please let me know your opinion.

**Comment by Gauravp-NEC** [comment]
> @amercader @Chealer ,Did the below changes looks better than commit [46e3754](https://github.com/ckan/ckan/commit/46e37544da99e085b1568ec13b1845e856107f73) ? If you agree I will commit the changes. Please let me know your opinion. Thankyou :)
> 
> image

@amercader , please let me know your opinion. Thankyou

**Pull request merged**